drivers/hmis/spec/requests/hmis/hmis_form_setup.rb

```diff
@@ -1,43 +1,6 @@
 RSpec.shared_context 'hmis form setup', shared_context: :metadata do
-  let!(:base_assessment_form_definition) do
-    definition = File.read('drivers/hmis/lib/form_data/assessments/base_assessment.json')
-    Hmis::Form::Definition.validate_json(JSON.parse(definition))
-    create :hmis_form_definition, identifier: 'assessment', role: 'INTAKE', definition: definition
-  end
-  let!(:client_form_definition) do
-    definition = File.read('drivers/hmis/lib/form_data/records/client.json')
-    Hmis::Form::Definition.validate_json(JSON.parse(definition))
-    create :hmis_form_definition, identifier: 'client', role: 'RECORD', definition: definition
-  end
-  let!(:organization_form_definition) do
-    definition = File.read('drivers/hmis/lib/form_data/records/organization.json')
-    Hmis::Form::Definition.validate_json(JSON.parse(definition))
-    create :hmis_form_definition, identifier: 'organization', role: 'RECORD', definition: definition
-  end
-  let!(:project_form_definition) do
-    definition = File.read('drivers/hmis/lib/form_data/records/project.json')
-    Hmis::Form::Definition.validate_json(JSON.parse(definition))
-    create :hmis_form_definition, identifier: 'project', role: 'RECORD', definition: definition
-  end
-  let!(:project_coc_form_definition) do
-    definition = File.read('drivers/hmis/lib/form_data/records/project_coc.json')
-    Hmis::Form::Definition.validate_json(JSON.parse(definition))
-    create :hmis_form_definition, identifier: 'project_coc', role: 'RECORD', definition: definition
-  end
-  let!(:funder_form_definition) do
-    definition = File.read('drivers/hmis/lib/form_data/records/funder.json')
-    Hmis::Form::Definition.validate_json(JSON.parse(definition))
-    create :hmis_form_definition, identifier: 'funder', role: 'RECORD', definition: definition
-  end
-  let!(:inventory_form_definition) do
-    definition = File.read('drivers/hmis/lib/form_data/records/inventory.json')
-    Hmis::Form::Definition.validate_json(JSON.parse(definition))
-    create :hmis_form_definition, identifier: 'inventory', role: 'RECORD', definition: definition
-  end
-  let!(:search_form_definition) do
-    definition = File.read('drivers/hmis/lib/form_data/records/search.json')
-    Hmis::Form::Definition.validate_json(JSON.parse(definition))
-    create :hmis_form_definition, identifier: 'search', role: 'RECORD', definition: definition
+  before(:all) do
+    system 'RAILS_ENV=test bin/rake driver:hmis:seed_definitions'
```

Comment:
Is this OK to do?

Reply:
That's probably fine, but it might be cleaner to:
```suggestion
    Rake::Task['driver:hmis:seed_definitions'].invoke
```

Reply:
I tried that but I had to run `Rails.application. load_tasks` for it to work, which I think was slow

Reply:
it's fine as is.